src/Store.h

```diff
@@ -150,11 +150,11 @@ class StoreEntry : public hash_link, public Packable
     void dump(int debug_lvl) const;
     void hashDelete();
     void hashInsert(const cache_key *);
-    void registerAbort(STABH * cb, void *);
+    void registerAbort(const AsyncCall::Pointer &);
```

Comment:
Since we are modifying these methods, lets find a better name and document them. I suggest:

```suggestion
    /// notify the StoreEntry writer of a 3rd-party initiated StoreEntry abort
    void registerAbortCallback(const AsyncCall::Pointer &);
```